### DISCUSSION TITLE
CornerRadius MainWindow

### DISCUSSION THREAD

**Discussion by ProgramistaZaDyszke** [OrgEleCho/EleCho.WpfSuite#6] [Ideas]
What about making corner rounded for main window? It is based on ChromeWindow so it should not be a problem... right? :)

**Comment by SlimeNull** [comment]
It only has an effect when Windows Aero (also called Desktop Window Manager) is not enabled or the [GlassFrameThickness](https://learn.microsoft.com/en-us/dotnet/api/system.windows.shell.windowchrome.glassframethickness?view=windowsdesktop-7.0) property is set to 0.

But if you want use WindowOption.Backdrop, [GlassFrameThickness](https://learn.microsoft.com/en-us/dotnet/api/system.windows.shell.windowchrome.glassframethickness?view=windowsdesktop-7.0) must be set to a non-zero value.
